src/select-field.jsx

```diff
@@ -82,7 +82,7 @@ let SelectField = React.createClass({
   },
 
   onChange(e, index, payload) {
-    if (payload) {
+    if (payload && this.props.valueMember) {
       e.target.value = payload[this.props.valueMember] || payload;
     }
```

Comment:
The problem was on these lines. Since the `SelectField` doesn't have a `valueMember` prop, the `payload` object was passed up in its entirety rather than the value. Checking for a `valueMember` solved the issue. 

However, I wonder if these three lines shouldn't be removed entirely since the same action is taken in `DropDownMenu` on [line 284](https://github.com/callemall/material-ui/blob/master/src/drop-down-menu.jsx#L284)? I also can't quite figure why the changes I made broke this. I think this might have been a hidden issue all along?